MSDF.StudentEngagement.Web/MSDF.StudentEngagement.Resources/Services/Encryption/EncryptionService.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Text.Encodings.Web;
+using Newtonsoft.Json;
+
+namespace MSDF.StudentEngagement.Resources.Services.Encryption
+{
+    public interface IEncryptionService
```

Comment:
Rename to Provider instead of Service. (The main idea is that Services use database calls. Providers modify or provide information. For eample: StudentService would connect to database, map and return student data. ImageProvider would calculate the student image base on their Id. Makes sense? Note that when you change to provider then we should manually register in IoC